.claude/skills/delegate/SKILL.md

```
@@ -0,0 +1,60 @@
+
+承認後、計画の各ステップを実装し**ステップごとに自動コミット**する。ユーザー確認は不要。
+
+### コミットルール
+
+- **1ステップ = 1コミット**（変更を必ずコミット単位で区切る）
+- `git add` は変更ファイルを明示的に指定する（`git add -A` 禁止）
+- コミット前に `pnpm lint` と `pnpm format:check` を実行する
+
+### コミットメッセージのフォーマット
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)